fiftyone/operators/executor.py

```diff
@@ -199,7 +200,7 @@ async def execute_or_delegate_operator(operator_uri, request_params):
         try:
             raw_result = await (
                 operator.execute(ctx)
-                if asyncio.iscoroutinefunction(operator.execute)
+                if inspect.isawaitable(operator.execute)
```

Comment:
This will always return false. These are not functionally equivelent.

Reply:
<img width="712" alt="image" src="https://github.com/voxel51/fiftyone/assets/462228/323c8c16-3184-4f48-a11a-3b43c0911912">

Reply:
<img width="792" alt="image" src="https://github.com/voxel51/fiftyone/assets/462228/49ee65d0-4f01-4bf5-8295-8e6f8e171651">